app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str):
+    name = command.split()
```

Comment:
The variable `name` is not very descriptive. Consider using a more descriptive name like `command_parts` or `file_names` to improve code readability.